client-spark/spark2/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -94,7 +94,7 @@ private void sendShuffleData(String taskId, List<ShuffleBlockInfo> shuffleDataIn
         for (ShuffleBlockInfo sbi : shuffleDataInfoList) {
           releaseSize += sbi.getFreeMemory();
         }
-        taskToBuffManager.get(taskId).freeAllocatedMemory(releaseSize);
+        taskToBufferManager.get(taskId).freeAllocatedMemory(releaseSize);
```

Comment:
It'd better add existence checking to avoid NPE.

Reply:
Sorry, I miss the comment, but in our logic, we can guarantee this taskId's existence.